src/View/View.php

```diff
@@ -1344,9 +1344,10 @@ public function pluginSplit(string $name, bool $fallback = true): array
     protected function _getLayoutFileName(?string $name = null): string
     {
         if ($name === null) {
+            /** @var string $name */
```

Comment:
Technically, it could be also the documented false when you submit it via setLayout before calling renderLayout.
I would prefer a clean exception handling in this case here, with meaningful message that this is bonkers to do as dev :)

Reply:
I added check for `$this->layout` being empty.